src/styles/home.css

```
@@ -56,4 +56,26 @@
 
 .testimonials-conteiner{
     margin: 3em 0em;
+}
+
+.home-conteiner {
+  background: #ffffff;
+  transition: 0.5s;
+}
+
+/* Dark theme color */
+.home-conteiner.dark {
+  background: #000000;
+}
+
+.dark .main-header {
```

These could be combine as you are applying just one and only style. 
You can just trail them one after another, separated by commas. 
```suggestion
.dark .main-header,  .dark .main-subheader, ... {
```

---

Done